filebeat/module/haproxy/log/ingest/pipeline.yml

```diff
@@ -31,7 +31,7 @@ processors:
     pattern_definitions:
       HAPROXY_DATE: (%{MONTHDAY}[/-]%{MONTH}[/-]%{YEAR}:%{HOUR}:%{MINUTE}:%{SECOND})|%{SYSLOGTIMESTAMP}
 - date:
-    if: ctx.event.timezone == null
```

Comment:
I think that this to be an issue, the `event` object has to be null, but I wonder how this can happen if filebeat modules are used, they always set values under `event`.
If `ctx.event` is not null, it doesn't matter if `timezone` is null, this expression should work.